gapis/api/vulkan/resources.go

```diff
@@ -835,8 +835,12 @@ func (s ShaderModuleObjectʳ) ResourceData(ctx context.Context, t *api.GlobalSta
 	if err != nil {
 		return nil, fmt.Errorf("Could not get resource data %v", err)
 	}
-	source := shadertools.DisassembleSpirvBinary(words)
-	return api.NewResourceData(&api.Shader{Type: api.ShaderType_Spirv, Source: source}), nil
+	source, sourceLanguage, isCross, err := shadertools.ExtractDebugSource(words)
+	if err != nil {
+		return nil, fmt.Errorf("Could not get decompile shader: %v", err)
```

Comment:
This should not be a fatal error. Do the same here as you did for the Pipelines below: log the error and return a SPIRV-only shader.